code/modules/paperwork/photography.dm

```diff
@@ -199,7 +215,13 @@
 		atoms.Add(the_turf);
 		// As well as anything that isn't invisible.
 		for(var/atom/A in the_turf)
-			if(A.invisibility) continue
+			if(A.invisibility )
+				if(see_ghosts && istype(A,/mob/dead/observer))
+					var/mob/dead/observer/O = A

```

Comment:
I think this is whats adding in the Icon images as it gets them @tigercat2000 this one seems to be taking whaterver the oberserver icon is then using the atoms.add bit...Maybe make it so it adds a Generic ghost sprite instead of the actual mob? This makes sense in my head..never does this end well.